src/Microsoft.Azure.WebJobs.ServiceBus/Config/ServiceBusOptionsFactory.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Azure.WebJobs.ServiceBus;
+using Microsoft.Extensions.Options;
+
+namespace WebJobs.ServiceBus.Config
+{
+    public class ServiceBusOptionsFactory : IConfigureOptions<ServiceBusOptions>
```

Comment:
Does this need to be public? Looks like it's internal impl. details for AddServiceBus

Reply:
It's actually not needed anymore; will move to this: https://twitter.com/davidfowl/status/1004248179628392448. Also cleaned up some if its usage in the tests.